packages/launcher/lib/detect.ts

```diff
@@ -16,26 +16,26 @@ import {
 } from './types'
 import * as windowsHelper from './windows'
 
-type HasVersion = {
-  version?: string
-  majorVersion?: string | number
+type HasVersion = Partial<FoundBrowser> & {
+  version: string
   name: string
 }
 
 export const setMajorVersion = <T extends HasVersion>(browser: T): T => {
-  let majorVersion = browser.majorVersion
-
-  if (browser.version) {
-    majorVersion = parseInt(browser.version.split('.')[0]) || browser.version
-    log(
-      'browser %s version %s major version %s',
-      browser.name,
-      browser.version,
-      majorVersion,
-    )
-  }
+  const majorVersion = parseInt(browser.version.split('.')[0]) || browser.version
+
+  const unsupportedVersion = browser.minSupportedVersion && majorVersion < browser.minSupportedVersion
+  const warning = unsupportedVersion ? `Cypress does not support running ${browser.displayName} version ${majorVersion} - it is too old. To use ${browser.displayName} with Cypress, install a version of ${browser.displayName} newer than or equal to ${browser.minSupportedVersion}.` : undefined
```

Comment:
"it is too old!!" haha, this phrase made me laugh. This phrase probably unnecessary and may be too broad in explaining why we may not support a certain browser. 

```suggestion
  const warning = unsupportedVersion ? `Cypress does not support running ${browser.displayName} version ${majorVersion}. To use ${browser.displayName} with Cypress, install a version of ${browser.displayName} newer than or equal to ${browser.minSupportedVersion}.` : undefined
```

Reply:
3c6d6d3